R/statDigitPref.R

```diff
@@ -0,0 +1,196 @@
+# ref.:
```

Comment:
Should I ignore this file? It looks like `statDigitPref_refactoring.R` is at least closer to package-format.